command/action.go

```diff
@@ -46,102 +45,98 @@ General Options:
 
 Action Specific Options:
 
-  -job <job-id>
+  -job=<job-id>
     Specifies the job in which the Action is defined
 
-  -allocation <allocation-id>
+  -alloc=<allocation-id>
```

Comment:
I don't think we put `=` anywhere else? I just spot-checked a bunch of commands and didn't find any that show this.

Reply:
Ah interesting, so every command is inconsistent because we do add the `=` in `generalOptionsUsage` 😅 
https://github.com/hashicorp/nomad/blob/4850f072955fee6ede30e9d17392e5f8dca20854/command/meta.go#L345-L413

I will remove them here and then maybe we can also remove from `generalOptionsUsage`?